designs/credentialManagement/credentialManagement.md

```diff
@@ -0,0 +1,139 @@
+# Credential Management
+
+## Abstract
+
+This document describes the process of managing and switching of AWS credentials within the toolkit.
+
+## Motivation
+
+AWS provides many different ways to retrieve the credentials required to make calls to AWS including but not limited to:
+credential/config files, environment variables, system properties, HTTP metadata APIs. A user may also assume roles using 
+a source set of credentials or use MFA. Due to this large matrix of possibilities the toolkit strives to provide an 
```

Comment:
grammar nit: this should be two sentences and inverted

Reply:
Broken up